### PR TITLE
extracting image library

### PR DESCRIPTION
**Pull request by lohanidamodar**

### CODE REVIEW

**Comment by eldadfux** [comment]
Tests are failing, seems like we need to update some paths that are not relevant once the tests are not running on Docker.

**Comment by lohanidamodar** [comment]
@eldadfux Php 7.4 is working but Php 8 and Nightly are failing. at (`pecl install imagick`).

**Comment by lohanidamodar** [comment]
@eldadfux I believe it's ready!

File: src/Resize/Resize.php [hunk 1]
```diff
@@ -0,0 +1,220 @@
+<?php
+
+namespace Utopia\Resize;
+
+use Exception;
+use Imagick;
+
+class Resize
```
Comment (eldadfux) [inline src/Resize/Resize.php:8]: We should probably change the name to `Image` as well

File: src/Resize/Resize.php [hunk 2]
```diff
@@ -0,0 +1,220 @@
+<?php
+
+namespace Utopia\Resize;
```
Comment (eldadfux) [inline src/Resize/Resize.php:3]: Change the namespace to `Image`

File: tests/Resize/ResizeTest.php [hunk 3]
```diff
@@ -0,0 +1,170 @@
+<?php
+
+namespace Appwrite\Tests;
+
+use Utopia\Resize\Resize;
+use PHPUnit\Framework\TestCase;
+
+class ResizeTest extends TestCase
```
Comment (eldadfux) [inline tests/Resize/ResizeTest.php:8]: Change name and namespaces

**Review by eldadfux — CHANGES_REQUESTED** [review]

**Comment by eldadfux** [comment]
@lohanidamodar please fix the psalm warnings and add a readme file with some examples 🙏🏻

**Comment by lohanidamodar** [comment]
> @lohanidamodar please fix the psalm warnings and add a readme file with some examples 🙏🏻

Sure

**Review by eldadfux — APPROVED** [review]

**Pull request merged**